server/lib/accounts.js

```diff
@@ -91,6 +118,28 @@ Accounts.onCreateUser(function(options, user = {}) {
 		}
 	}
 
+	if (!user.active) {
+		const destinations = [];
+		let email = {};
+
+		RocketChat.models.Roles.findUsersInRole('admin').forEach(function(adminUser) {
+			if (adminUser.emails[0]) {
```

Comment:
Please use `if (adminUser.emails && adminUser.emails[0] && adminUser.emails[0].address)` to make sure there's an `emails` key and an address on user emails.